tasks/install-matlab/v0/test/suite.ts

```diff
@@ -1,79 +1,67 @@
-// Copyright 2020-2022 The MathWorks, Inc.
```

Comment:
2024

Reply:
Oh no, I've been still thinking it's 2023 😵